core/triggers/interface_98_modorderfromsupplierordermulticompany_Orderfromsupplierordermulticompanytrigger.class.php

```diff
@@ -410,7 +410,7 @@ public function run_trigger($action, $object, $user, $langs, $conf)
 				$fkSupplierOrder = array_shift($object->linkedObjectsIds['commandefourn']);
 				$supplierOrder = new CommandeFournisseur($object->db);
 				$supplierOrder->fetch($fkSupplierOrder);
-				$supplierOrder->set_date_livraison($user, $object->date_livraison);
+				$supplierOrder->setDeliveryDate($user, $object->date_livraison);
```

Comment:
using setDeliveryDate will loss Dolibarr < V14 compatibility use is_callable for compatibility
exemple : 
```suggestion
				if (is_callable(array($supplierOrder, 'setDeliveryDate'))) {
					$supplierOrder->setDeliveryDate($user, $object->date_livraison);
				} else {
					// For Dolibarr < V14
					$supplierOrder->set_date_livraison($user, $object->date_livraison);
				}
```